_events/2022/2022-03-dei-speaker-series.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Allies, Collaboration & Automation - Working towards inclusive Engagement in Research Computing
+subtitle: US-RSE DEI-WG Speaker Series
+expires: 2021-03-11
+event_date: "March 11, 2021"
+layout: event
+duration: 60
+repeated: false
+category: dei
+time:
+    - - start: 2021-03-11T19:00:00Z
+        end: 2021-03-11T20:00:00Z
+---
+
+
+## US-RSE DEI-WG Speaker Series Info
+
+The US-RSE DEI-WG is working to bring to the US-RSE community quarterly visits from RSE professionals with vested interests or research into DEI in the RSE community at large. Our community pledges to continually educate ourselves through research on best practices focused on diversity, equity, and inclusion. The US-RSE is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community. We encourage everyone to suggest speakers and topics and are open to different formats and timelines of the events. Feedback is much appreciated.
+
+
+## Event
+
+The US-RSE Diversity, Equity, and Inclusion Working Group (DEI-WG) is proud to
+present this speaker series event in conjunction with the international DiveRSE initiative
```

Comment:
Is there a link we can add to DiveRSE so a reader like myself that wants to know more can easily click?

Reply:
There isn't one yet I don't believe!

Reply:
oh wow! Do you know who is running it so I can ping them to nudge to make a little page somewhere?

Reply:
"DiveRSE" is something that grew out of the online component of the [Lorentz workshop Vive la différence - research software engineers](https://www.researchsoft.org/lorentz/). I don't know of any public information available about that yet either, but perhaps we could just say "the online component of the Lorentz workshop..." and update to include "DiveRSE" if that does in fact become a reality?

Reply:
Or, looking at the date, just leave it off and put the page up. We need to start advertising this ASAP.